src/sentry/conf/server.py

```diff
@@ -780,6 +780,7 @@ def create_partitioned_queues(name):
 REST_FRAMEWORK = {
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
     'DEFAULT_PERMISSION_CLASSES': ('sentry.api.permissions.NoPermission', ),
+    'EXCEPTION_HANDLER': 'sentry.api.handlers.custom_exception_handler',
```

Comment:
Our endpoints also have a `handle_exception` hook method that you could use to convert the RateLimitExceeded exception into a response.

Reply:
Yeah I was considering that too, I wasn't sure which would be preferable. I ended up using the handler because it means it'll apply to any DRF view rather than just our api endpoints. Do you have a strong preference? I don't mind changing if so.

Reply:
I don't have strong opinions. We only implement `handle_exception` on a handful of endpoints right now.